adflow/pyADflow.py

```diff
@@ -4516,259 +4476,244 @@ def setOption(self, name, value):
-            'anknsubiterturb':[int,1],
-            'ankturbkspdebug':[bool,False],
-            'ankusematrixfree':[bool,True],
+            'useANKSolver':[bool, False],
+            'ANKUseTurbDADI':[bool, True],
+            'ANKSwitchTol':[float, 1.0],
+            'ANKSubspaceSize':[int, -1],
+            'ANKMaxIter':[int, 40],
+            'ANKLinearSolveTol':[float, 0.05],
+            'ANKLinResMax':[float, 0.9],
+            'ANKASMOverlap':[int, 1],
+            'ANKPCILUFill':[int, 2],
+            'ANKJacobianLag':[int, 10],
+            'ANKInnerPreconIts':[int, 1],
+            'ANKOuterPreconIts':[int, 1],
+            'ANKCFL0':[float, 5.0],
+            'ANKCFLMin':[float,1.0],
+            'ANKCFLLimit':[float, 1e5],
+            'ANKCFLFactor':[float, 10.0],
+            'ANKCFLExponent':[float, 0.5],
+            'ANKCFLCutback':[float,0.5],
+            'ANKStepFactor':[float, 1.0],
+            'ANKStepMin':[float, 0.01],
+            'ANKConstCFLStep':[float, 0.4],
+            'ANKPhysicalLSTol':[float, 0.2],
+            'ANKPhysicalLSTolTurb':[float, 0.99],
+            'ANKUnsteadyLSTol':[float, 1.0],
+            'ANKSecondOrdSwitchTol':[float, 1e-16],
+            'ANKCoupledSwitchTol':[float, 1e-16],
+            'ANKTurbCFLScale' : [float, 1.0],
+            'ANKUseFullVisc' : [bool, True],
+            'ANKPCUpdateTol':[float,0.5],
+            'ANKADPC':[bool, False],
+            'ANKNSubiterTurb':[int,1],
+            'ANKTurbKSPDebug':[bool,False],
+            'ANKUseMatrixFree':[bool,True],
 
             # Load Balance/partitioning parameters
-            'blocksplitting':[bool, True],
-            'loadimbalance':[float, 0.1],
-            'loadbalanceiter':[int, 10],
-            'partitiononly':[bool, False],
-            'partitionlikenproc':[int, -1],
+            'blockSplitting':[bool, True],
+            'loadImbalance':[float, 0.1],
+            'loadBalanceIter':[int, 10],
+            'partitionOnly':[bool, False],
+            'partitionLikeNProc':[int, -1],
 
             # Misc Parameters
-            'autosolveretry':[bool, False],
-            'autoadjointretry':[bool, False],
-            'numbersolutions':[bool, True],
-            'printiterations':[bool, True],
-            'printtiming':[bool, True],
-            'setmonitor':[bool, True],
-            'printwarnings':[bool, True],
-            'monitorvariables':[list, ['cpu','resrho', 'resturb', 'cl', 'cd']],
-            'surfacevariables':[list, ['cp','vx', 'vy','vz', 'mach']],
-            'volumevariables':[list, ['resrho']],
-
-            # Multidisciplinary Coupling Parameters:
-            'forcesastractions':[bool, True],
+            'numberSolutions':[bool, True],
+            'printIterations':[bool, True],
+            'printTiming':[bool, True],
+            'setMonitor':[bool, True],
+            'printWarnings':[bool, True],
+            'monitorVariables':[list, ['cpu','resrho', 'resturb', 'cl', 'cd']],
```

Comment:
Should we also update these options? It should be `resRho` etc. with the current convention right? I think these are more commonly used compared to other affected options, so I am also fine with not changing these.

Reply:
I suggest that we keep this for now. In the future, I plan to change the way these are set up in baseclasses. Like the list of choices, there should be a _set_ of all possible variables to monitor, and the user can specify a list (does the order matter? maybe for print out?) and we check in base classes if that is a subset of the total allowable set of monitor variables.